shared/src/ClientPropertyCollection.cs

```diff
@@ -2,12 +2,10 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
-using System.Collections;
 using System.Collections.Generic;
 using Microsoft.Azure.Devices.Shared;
-using Newtonsoft.Json.Linq;
 
-namespace Microsoft.Azure.Devices.Client
+namespace Microsoft.Azure.Devices.Shared
```

Comment:
Back to the conversation earlier, does it make more sense to have this inside of the Client class? My thought is no since we can use this as part of the ModuleClient.

Reply:
to put the client property collection in device client class? why?

Reply:
No reason, just wanted to make sure we're aligned.